lib/plottr_components/src/components/timeline/BeatTitleCell.js

```diff
@@ -154,9 +158,15 @@ const BeatTitleCellConnector = (connector) => {
       if (event.which === 27) this.setState({ editing: false })
     }
 
+    handleSetDraggedBeat = beat => {
```

Comment:
This is unnecessary lambda wrapping.
You can use `this.props.setDraggedBeat` wherever you were using this function instead.